src/js/view/title.ts

```diff
@@ -3,25 +3,40 @@ import {
     replaceInnerHtml
 } from 'utils/dom';
 import { Browser } from 'environment/environment';
+import type { PlaylistFeedItemWithMedia } from 'parsers/jwparser';
+import type { PlaylistMRSSItemWithMedia } from 'parsers/mediaparser';
+import type ViewModel from './view-model';
+import type { PlayerViewModel } from './view-model';
+import type Model from 'controller/model';
 
-const Title = function(_model) {
-    this.model = _model.player;
-    this.truncated = _model.get('__ab_truncated') && !Browser.ie;
-};
+class Title {
+    model: PlayerViewModel;
```

Comment:
👍 We don't display ad titles, only the playlist item title so use the player properties to ignore change events from the view model when we switch to "instream" (ads) mode.